runtime/src/lib.rs

```diff
@@ -373,22 +373,22 @@ parameter_types! {
     pub const ProposalBond: Permill = Permill::from_percent(5);
     pub ProposalBondMinimum: Balance = 10 * DOLLARS;
     pub ProposalBondMaximum: Balance = 50 * DOLLARS;
-    pub const SpendPeriod: BlockNumber = 30 * DAYS;
+    pub const SpendPeriod: BlockNumber = conf!(mainnet: 30 * DAYS, testnet: 30 * DAYS);
     pub const Burn: Permill = Permill::from_percent(1);
 
     pub const TipCountdown: BlockNumber = 2 * DAYS;
     pub const TipFindersFee: Percent = Percent::from_percent(5);
     pub TipReportDepositBase: Balance = deposit(1, 0);
     pub BountyDepositBase: Balance = deposit(1, 0);
-    pub const BountyDepositPayoutDelay: BlockNumber = 6 * DAYS;
-    pub const BountyUpdatePeriod: BlockNumber = 35 * DAYS;
+    pub const BountyDepositPayoutDelay: BlockNumber = conf!(mainnet: 6 * DAYS, testnet: 6 * DAYS);
+    pub const BountyUpdatePeriod: BlockNumber = conf!(mainnet: 35 * DAYS, testnet: 35 * DAYS);
```

Comment:
same as for the previous

Reply:
and there are others below

Reply:
Yes, this happened due to the fact that at the moment there are questions about the configs on the mainnet. 
Updated the configs by deleting the conf! in places where it is not needed now.